airgun/widgets.py

```diff
@@ -601,7 +601,6 @@ def read(self):
         """Returns a list of available actions."""
         return self.items
 
-
```

Comment:
That's a bad idea as it will cause pep8 checks to fail :) there should be 2 lines between classes, please revert this change.